client/scss/layouts/_home.scss

```diff
@@ -1,30 +1,6 @@
-.homepage header {
-  @apply w-text-primary;
-
-  display: flex;
-  flex-flow: row nowrap;
-  gap: 1em;
-
-  .col1 {
-    width: 50px;
-    margin-inline-end: 1em;
-    padding: 0;
-
-    // make way for the nav-menu button on mobile
-    @include media-breakpoint-down(xs) {
-      position: relative;
-      inset-inline-start: $mobile-nav-indent;
-    }
-  }
-
-  .avatar {
-    margin-top: 0.5em;
-    flex-shrink: 0;
-  }
-
-  .user-name {
-    font-size: 1.3em;
-    font-weight: 600;
+.homepage {
+  h1 {
```

Comment:
I recommend we put these styles in the `header`. while there should only be 1 h1 - better to be safe.

We should also use the `header__title` class instead of styling on the `h1` directly.

Reply:
done